common/djangoapps/student/models.py

```diff
@@ -280,13 +294,36 @@ class Meta(object):
         # Translators: 'Other' refers to the student's level of education
         ('other', ugettext_noop("Other education"))
     )
+
+    PROFESSION_CHOICES = (
+        ('media', ugettext_noop('Media professional, activist, human rights defender')),
+        ('other', ugettext_noop("Other")),
+        ('undisclosed', ugettext_noop("Undisclosed")),
```

Comment:
I would decrease the key for `undisclosed` choice to 5 chars and also decrease max_length